[vcpkg] vcpkg_acquire_msys wrong working dir at line 124

```
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:50 (_execute_process):
  _execute_process called with no value for WORKING_DIRECTORY.
Call Stack (most recent call first):
  scripts/cmake/vcpkg_acquire_msys.cmake:121 (vcpkg_execute_required_process)
  ports/lapack-reference/portfile.cmake:17 (vcpkg_acquire_msys)
  scripts/ports.cmake:90 (include)
```

Uses` ${SOURCE_PATH}` at line 124 which is not necessarily defined in the function call

--- a/scripts/cmake/vcpkg_acquire_msys.cmake
+++ b/scripts/cmake/vcpkg_acquire_msys.cmake
@@ -121,7 +121,7 @@ function(vcpkg_acquire_msys PATH_TO_ROOT_OUT)
       vcpkg_execute_required_process(
           ALLOW_IN_DOWNLOAD_MODE
           COMMAND TASKKILL /F /IM gpg-agent.exe /fi "memusage gt 2"
-          WORKING_DIRECTORY ${SOURCE_PATH}
+          WORKING_DIRECTORY ${TOOLPATH}
       )
   endif()
 

